Detectors/TRD/workflow/src/TRDTrapSimulatorSpec.cxx

```diff
@@ -661,7 +597,8 @@ void TRDDPLTrapSimulatorTask::run(o2::framework::ProcessingContext& pc)
     LOG(info) << "Raw data words written = " << mTotalRawWordsWritten << " with a vector size = " << rawdata.size();
   }
   LOG(debug) << "END OF RUN .............";
-
+  //TODO does anyone care to have the digits to tracklet mapping. Do we then presort the digits inline with the index or send both digits and sorted index.
```

Comment:
Good question. One could get very close by just knowing the MCM and the tracklet position - if more is needed, we can get it from the debug stream.